src/main/kotlin/gdx/liftoff/data/project/Data.kt

```diff
@@ -38,6 +37,16 @@ data class AdvancedProjectData(
    */
   val androidSdkVersion: String = "35"
 
+  /**
+   * Should match the recommended/tested by the libGDX version.
+   */
+  val androidPluginVersion: String = "8.6.1"
+
+  /**
+   * Will be set to this value if using a Java version of at least 8.
+   */
```

Comment:
I think this comment is no longer correct, since there's no dynamic setting anymore (nor ability to set Java version 7), but it isn't really a problem. End users won't see this comment and I can change it just after I merge.

Reply:
At the moment it's still only added if Java version is not 6 and 7 but I agree that logic can be removed.